Sources/Common/Core/Math/index.js

```diff
@@ -2146,6 +2146,51 @@ export function getMajorAxisIndex(vector) {
   return axisIndex;
 }
 
+// This function iteratively associate a column with a row by choosing
+// the greatest absolute value from the remaining row and columns
+// Returns an array L such as if the matrix is column major,
+// the row i is associated with column L[i]
+export function getRowColumnAssociations(matrix, n = 3) {
```

Comment:
The documentation is not clear to me. 
It seems that you return an order for columns. What is this order ?

Reply:
The order of the column indices is the order of the rows, this is what I mean by `the row i is associated with column L[i]`
How would you phrase this differently?